modAionImpl/test/org/aion/zero/impl/db/AionRepositoryImplTest.java

```diff
@@ -146,7 +146,7 @@ public void testAccountStateUpdateStorageRow() {
         track.flush();
 
         byte[] retrievedValue =
-                repository.getStorageValue(defaultAccount, new DataWord(key)).getNoLeadZeroesData();
+                repository.getStorageValue(defaultAccount, new DataWord(key)).get().getNoLeadZeroesData();
```

Comment:
what if the getStorageValue return Optional.empty() ?

Reply:
I'll update these tests so that if empty is returned they fail